server/src/main/java/org/apache/iotdb/db/metadata/mtree/MTreeBelowSGMemoryImpl.java

```diff
@@ -384,7 +384,7 @@ private IMemMNode checkAndAutoCreateDeviceNode(String deviceName, IMemMNode devi
       if (!DataNodeSpaceQuotaManager.getInstance().checkDeviceLimit(storageGroupMNode.getName())) {
         throw new ExceedQuotaException(
             "The number of devices has reached the upper limit",
-            TSStatusCode.EXCEED_QUOTA_ERROR.getStatusCode());
+            TSStatusCode.SPACE_QUOTA_EXCEEDED.getStatusCode());
       }
```

Comment:
as we have the `quota_enable` configuration, I think we can check the parameter first when we check the space/throttle quota.
And I see there are two places that contains the `quota_enable` configurtion, the first is in the CommonDescriptor, the second is in the  IoTDBDescriptor, is it really needed in two places？